lib/agent.js

```diff
@@ -44,19 +44,24 @@ function Agent(opts) {
   if (!opts.type)
     opts.type = 'udp4'
 
+  if (opts.socket) {
+    delete opts.port;
```

Comment:
Project code style [CS] does not use ";" as statement separator, so it should be removed